Fabric.Authorization.API/scripts/Install-Authorization-Utilities.psm1

```diff
@@ -385,6 +411,27 @@ function Add-ListOfUsersToDosAdminGroup($edwAdminUsers, $connString, $authorizat
     }	
 }
 
+function Add-AccountToEDWAdmin($accountName, $domain, $connString) {
+	$samAccountName = Get-SamAccountFromAccountName -accountName $accountName
+	if ((Test-IsUser -samAccountName $samAccountName -domain $domain) -or (Test-IsGroup -samAccountName $samAccountName -domain $domain)) {
+		try {
+			
+			Add-UserOrGroupToEdwAdmin -userOrGroup $accountName -connString $connString
+		}
+		catch {
+            $exception = $_.Exception
+            if ($exception.Response -ne $null) {

```

Comment:
The exception in this case will never have a Response object, that only exists on System.Net.WebException I believe, which won't be thrown here.

Reply:
Ah, good catch.  I was wondering why the exceptions didnt make sense

Reply:
Removed the exception altogether.  No need to catch it.

It was a case of copy pasta